brainsets/processing/signal.py

```
@@ -167,3 +167,12 @@ def cube_to_long(
     )
 
     return trials, units
+
+
+_functions = [
+    "downsample_wideband",
+    "extract_bands",
+    "cube_to_long",
+]
+
+__all__ = _functions
```

Ruff (0.15.7)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)